src/command.js

```diff
@@ -58,7 +57,7 @@ class Command {
   sanitizePackageNames(packageNames) {
     packageNames ??= [];
     packageNames = packageNames.map(packageName => packageName.trim());
-    return _.compact(_.uniq(packageNames));
+    return Array.from(new Set(packageNames)).filter(Boolean);
```

Comment:
It would be hard to object against a banal comment for some logic that will probably never change but still, I do think that this concern is overreaching. I specifically chose `.filter(Boolean)` because I felt that was the clearest way to do it - way clearer than the misleading `_.compact` itself - and I also think it's common knowledge that you turn something to a set and then back in order to eliminate duplicates. Of all the ppm code, I definitely wouldn't be the most concerned about this...